third_party/move/tools/move-formatter/tests/formatter/spec_struct/input3.fmt.move

```diff
@@ -0,0 +1,23 @@
+/// test_point: has ability{'copy', 'drop', 'store'} with comment
+
+/// The `ASCII` module defines basic string and char newtypes in Move that verify
+/// that characters are valid ASCII, and that strings consist of only valid ASCII characters.
+module std::ascii {
+    use std::vector;
+    use std::option::{Self, Option};
+
+    /// The `String` struct holds a vector of bytes that all represent
+    /// valid ASCII characters. Note that these ASCII characters may not all
+    /// be printable. To determine if a `String` contains only "printable"
+    /// characters you should use the `all_characters_printable` predicate
+    /// defined in this module.
+    struct String has /*comment*/ copy, drop /*comment*/, store /*comment*/ {
+        // comment
+        bytes: vector<u8>, // comment
+
```

Comment:
not sure whether we should remove this empty line? cc @vineethk

Reply:
Yes, there seems to be an issue in formatting structs in general and new lines:

```
struct Blah {

    x: u8,

    y: u64,

    z: u64

}
```

The above will be left unchanged by the formatter.

Reply:
after formatted, it should like:
```
struct Blah {
    x: u8,
    y: u64,
    z: u64
}
```
?